wolfcrypt/src/port/Renesas/renesas_sce_rsa.c

```diff
@@ -0,0 +1,430 @@
+                info->flags2.bits.rsapub1024_installedkey_set = 1;
+            }
+            else if (size == 2048) {
+                if (info->sce_wrapped_key_rsapri2048 != NULL) {
+                    XFREE(info->sce_wrapped_key_rsapri2048, NULL, 
+                                    DYNAMIC_TYPE_RSA_BUFFER);
+                }
+                if (info->sce_wrapped_key_rsapub2048 != NULL) {
+                    XFREE(info->sce_wrapped_key_rsapub2048, NULL, 
+                                    DYNAMIC_TYPE_RSA_BUFFER);
+                }
+                info->sce_wrapped_key_rsapri2048 = 
+                (sce_rsa2048_private_wrapped_key_t*)XMALLOC(
+                    sizeof(sce_rsa2048_private_wrapped_key_t), NULL, 
+                                    DYNAMIC_TYPE_RSA_BUFFER);
+                    
+                if (info->sce_wrapped_key_rsapri2048 == NULL) {
+                    XFREE(wrapped_pair2048_key, 0, DYNAMIC_TYPE_RSA_BUFFER);
+                    return MEMORY_E;
+                }
+                
+                info->sce_wrapped_key_rsapub2048 =
+                (sce_rsa2048_public_wrapped_key_t*)XMALLOC(
+                    sizeof(sce_rsa2048_public_wrapped_key_t), NULL, 
+                                    DYNAMIC_TYPE_RSA_BUFFER);
+                    
+                if (info->sce_wrapped_key_rsapub2048 == NULL) {
+                    XFREE(wrapped_pair2048_key, 0, DYNAMIC_TYPE_RSA_BUFFER);
+                    XFREE(info->sce_wrapped_key_rsapub1024, 0, 
+                                    DYNAMIC_TYPE_RSA_BUFFER);
+                    return MEMORY_E;
+                }
+                /* copy generated key pair and free malloced key */
+                XMEMCPY(info->sce_wrapped_key_rsapri2048, 
+                            &wrapped_pair2048_key->priv_key,
+                            sizeof(sce_rsa2048_private_wrapped_key_t));
+                XMEMCPY(info->sce_wrapped_key_rsapub2048, 
+                            &wrapped_pair2048_key->pub_key,
+                            sizeof(sce_rsa2048_public_wrapped_key_t));
+                XFREE(wrapped_pair2048_key, 0, DYNAMIC_TYPE_RSA_BUFFER);
+                
+                info->flags2.bits.rsapri2048_installedkey_set = 1;
+                info->flags2.bits.rsapub2048_installedkey_set = 1;
+                
+            }
+        }
+        else {
+            WOLFSSL_MSG("Failed to generate key pair by SCE");
+            return CRYPTOCB_UNAVAILABLE;
+        }
+        
+        wc_sce_hw_unlock();
+    }
+}
+
+/* Perform rsa encryption/decryption by SCE
+ *
+ * in     Buffer to hold plain text
+ * inLen  Length of plain text in bytes
+ * out    Buffer to hold cipher text
+ * outLen Length of cipher in bytes
+ * key    Rsa key object
+ * rng    rng object
+ * ctx    Callback context
+ * return FSP_SUCCESS(0) on Success, otherwise negative value
+ */
+WOLFSSL_LOCAL int  wc_sce_RsaFunction(const byte* in, word32 inLen, byte* out,
```

Comment:
Spacing - reduce from two spaces to one after `int`.

Also check other functions in this file for similar spacing issue, thanks.

Reply:
Thanks! Fixed.